api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -61,9 +62,8 @@
   // If we later decide to tune this value, consider moving to the WorkbenchConfig.
   private static final int MAX_NOTEBOOK_SIZE_MB = 100;
   // "directory" for notebooks, within the workspace cloud storage bucket.
-  private static final String NOTEBOOKS_WORKSPACE_DIRECTORY = "notebooks";
-  private static final Pattern NOTEBOOK_PATTERN =
-      Pattern.compile(NOTEBOOKS_WORKSPACE_DIRECTORY + "/[^/]+(\\.(?i)(ipynb))$");
+  private static final String NOTEBOOKS_WORKSPACE_DIRECTORY = NotebooksServiceImpl.NOTEBOOKS_WORKSPACE_DIRECTORY;
```

Comment:
Can you define these constants on the interface instead? Odd to hav ea direct impl reference here